types/brainhubeu__react-carousel/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for @brainhubeu/react-carousel 1.10
+// Project: https://github.com/brainhubeu/react-carousel
+// Definitions by: Jack Allen <https://github.com/jackall3n>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as React from "react";
+
+type ImgProps = React.ReactComponentElement<'img'>;
+
+export interface DotsProps {
+    number?: number;
+    thumbnails?: ImgProps[];
+    value?: number;
+    onChange?(value: number): void;
+}
+
+export class Dots extends React.Component<DotsProps> {
+}
+
+export interface CarouselProps {
+    value?: number;
+    onChange?(value: number): void;
+    slides?: ImgProps[];
+    slidesPerPage?: number;
+    slidesPerScroll?: number;
+    itemWidth?: number;
+    offset?: number;
+    arrows?: boolean;
+    arrowLeft?: JSX.Element;
+    arrowRight?: JSX.Element;
+    addArrowClickHandler?: boolean;
+    autoPlay?: number;
+    stopAutoPlayOnHover?: boolean;
+    clickToChange?: boolean;
+    centered?: boolean;
+    infinite?: boolean;
+    draggable?: boolean;
+    keepDirectionWhenDragging?: boolean;
+    animationSpeed?: number;
+    dots?: boolean;
+}
+
+export class Carousel extends React.Component<CarouselProps> {
```

Comment:
This class isn't a named export of the module:
```
> require('@brainhubeu/react-carousel');
{ Dots: [Function: t], default: [Function: t] }
```

```suggestion
export default class extends React.Component<CarouselProps> {
```

Reply:
Agreed, changes made.